solr/solrj/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -131,6 +132,19 @@
 
   private final Runnable securityNodeListener;
 
+  private Map<String, CollectionWatch> collectionWatches = new ConcurrentHashMap<>();
```

Comment:
The reason I made a comment about using the concrete type here is that it makes it much easier to work with as a developer.  When you can see that the static type of this variable is ConcurrentHashMap, that helps you evaluate the code that touches it.

For example, when you use IDE features to 'click through' a method call or view the javadoc on a called method, you get the ConcurrentHashMap version of the javadoc/method instead of the Map version, which helps you more easily evaluate the correctness.